.github/scripts/commit_release.sh

```diff
@@ -7,11 +7,11 @@ git checkout -b "release/$1"
 # Update version in podspec.
 # (Search podspec for `.version = '1.2.3` and update with new version
 # number passed in as script argument).
-sed -i.bak -E "s/\.version *= *(["'"'"'])[0-9]\.[0-9]\.[0-9]["'"'"']/.version = \1$1\1/g" Thumbprint.podspec
+sed -i.bak -E "s/\.version *= *(["'"'"'])[0-9](\.[0-9])?(\.[0-9])?["'"'"']/.version = \1$1\1/g" Thumbprint.podspec
```

Comment:
Does this allow for, say, `10.11.4`? My regex-fu is really rusty...

Reply:
Good catch, no. I'll update in a separate PR

Reply:
https://github.com/thumbtack/thumbprint-ios/pull/38